ui/components/app/flask/snaps-authorship-pill/snaps-authorship-pill.js

```diff
@@ -13,7 +13,7 @@ import { useI18nContext } from '../../../../hooks/useI18nContext';
 
 const snapIdPrefixes = ['npm:', 'local:'];
 
-const SnapsAuthorshipPill = ({ snapId, version, className }) => {
+const SnapsAuthorshipPill = ({ snapId = '', version, className }) => {
```

Comment:
Why is this needed?

Reply:
Leads to a type error when trying to access undefined for `.startsWith` and `.replace`, but I suppose I can just use optional chaining.

Reply:
But it should be defined, no?

Reply:
It should, but it throws if you don't. Specifically, when you hit cancel or approve.